package.json

```diff
@@ -50,30 +50,39 @@
     "node": ">=12.14.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.1.2",
+    "@grpc/grpc-js": "^1.1.7",
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "bluebird": "^3.7.2",
     "chalk": "^4.0.0",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
+    "connect": "^3.7.0",
     "encryptedfs": "^1.0.2",
+    "get-port": "^5.1.1",
     "google-protobuf": "^4.0.0-rc.2",
+    "graphql": "^15.3.0",
     "isomorphic-git": "^1.7.4",
     "kbpgp": "^2.0.82",
     "node-forge": "^0.9.1",
     "pako": "^1.0.11",
+    "portable-fetch": "^3.0.0",
```

Comment:
What's this for?

Reply:
This was for the generated client in the tests folder, the whole generated client has been replaced by just a call to `http.request` so this has also been removed.